health.go

```diff
@@ -81,6 +88,7 @@ type (
 func New(opts ...Option) (*Health, error) {
 	h := &Health{
 		checks: make(map[string]Config),
+		tp:     trace.NoopTracerProvider(),
```

Comment:
is this right? no-op?

Reply:
yes, this is default value, real tracer can be set with the option